### PR TITLE
Fixing error in Read.me file 

### PR DESCRIPTION
**Pull request by Ritika-Agrawal811**
## Related Issue
#18 
Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
I have fixed issue 18 and made the necessary changes mentioned below.
## Proposed Changes

- removed java logo and added flask logo
- corrected the rendering of links for html , flask , css , bootstrap and javascript.

## Check List

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if appropriate):

Original             |  Updated
:-------------------------:|:-------------------------:
  |

### CODE REVIEW

**Comment by sameersrivastava13** [comment]
@Sejal2801 Can you please review once whether Java logo is removed or not? Because as I can see its still there.

File: README.md [hunk 1]
```diff
@@ -23,13 +23,14 @@ The data is sourced on a real-time basis from the CoWIN API, which is the offici
...
-<img height="200" width ="200" src = "https://github.com/Tejas1510/Hacking-Scripts/blob/main/assets/javalogo.png"></img>
+<img height="200" width ="400" src = "https://github.com/pallets/flask/blob/main/artwork/logo-full.svg"></img>
 <img height="200" width ="400" src = "https://github.com/Tejas1510/Hacking-Scripts/blob/main/assets/htmllogo.jfif"></img>
-</p>* [Flask](https://github.com/pallets/flask)
-* [HTML](https://www.w3.org/TR/html52/)
-* [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
-* [Bootstrap](https://getbootstrap.com/)
-* [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+</p>
+*<a href="https://github.com/pallets/flask">Flask</a>
```
Comment (sameersrivastava13) [inline README.md]: @Ritika-Agrawal811 It will good if you can add line break after every technology that you have mentioned.

File: README.md [hunk 2]
```diff
@@ -23,13 +23,14 @@ The data is sourced on a real-time basis from the CoWIN API, which is the offici
...
+*<a href="https://www.w3.org/TR/html52">HTML</a>
+*<a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a>
```
Comment (sameersrivastava13) [inline README.md]: @Ritika-Agrawal811 Line break here.

File: README.md [hunk 3]
```diff
@@ -23,13 +23,14 @@ The data is sourced on a real-time basis from the CoWIN API, which is the offici
...
+*<a href="https://getbootstrap.com">Bootstrap</a>
```
Comment (sameersrivastava13) [inline README.md]: @Ritika-Agrawal811 Similarly here.

**Comment by Ritika-Agrawal811** [comment]
I have created a branch of my name and then made the changes in that branch. 

In this below link you can see the changes:-
https://github.com/Ritika-Agrawal811/Vaccine-Tracker-App/tree/Ritika-Agrawal811

**Review by sameersrivastava13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Ritika-Agrawal811) [reply README.md]: I added the line breaks.

File: README.md (hunk 1 above)
Comment (Ritika-Agrawal811) [reply README.md]: I have added the line breaks after every technology.

**Pull request merged**